appformer-kogito-bridge/src/main/java/org/appformer/kogito/bridge/client/resource/impl/ResourceContentServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.bridge.client.resource.impl;
+
+import elemental2.promise.Promise;
+import org.appformer.kogito.bridge.client.resource.ResourceContentService;
+import org.appformer.kogito.bridge.client.resource.interop.ResourceContentEditorServiceWrapper;
+
+/**
+ *  A {@link ResourceContentService} implementation used when envelope API is available
+ *
+ */
+public class ResourceContentServiceImpl implements ResourceContentService {
```

Comment:
Minor comment too, and similar comment as above... what does a "XXXSerivceImpl" means? What is the default "impl"?   For me the classname does not properly refelcts which / what does this impl do....

Reply:
Changed to EnvelopeResourceContent. It won't be used directly, API users will have only to inject ResourceContentService